www/src/components/ColorModePicker.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+
+const storageKey = 'recharts-color-mode';
```

Comment:
nit nit: since this is a constant would be nice to make it upper case `STORAGE_KEY`